README.md

```diff
@@ -4,6 +4,15 @@ A project based learning activity for people who are getting started with Git an
 
 You can play the game at: https://githubschool.github.io/github-games/
 
+### Instructions for playing the game
+
+1. Press the space bar to begin.
+2. Use the up and down arrow keys to rotate the shape.
```

Comment:
Thank you for adding the instructions! I tested the game in my browser, but the down arrow key **_doesn't_** rotate the shape. Could you please confirm if that's the intended behavior?